prow/plugins.yaml

```diff
@@ -1,29 +1,10 @@
 plugins:
-  kyma-project/kyma:
-    - cat
-    - trigger
-  kyma-project/console:
-    - cat
-    - trigger
-  kyma-project/examples:
-    - cat
-    - trigger
-  kyma-project/test-infra:
-    - config-updater
-    - cat
-    - trigger
-  kyma-project/bundles:
-    - cat
-    - trigger
-  kyma-project/community:
-    - cat
-    - trigger
-  kyma-project/website:
-    - cat
-    - trigger
-  kyma-project/luigi:
+  kyma-project:
     - cat
     - trigger
+    - owners-label
+    - verify-owners
+    - approve
```

Comment:
If we are modifying the plugin scope to be organisation wide, I'd just suggest adding the `welcome`, `lgtm` and `size` plugins to the list

Reply:
I decided to not add any new plugin. I just change to scope to the organization level to make `plugins.yaml` file smaller.